app-conf/compile.conf

```diff
@@ -1,3 +1,3 @@
-hadoop_version=2.3.0
+hadoop_version=2.4.1
```

Comment:
It is just a conf I used while testing. Shouldn't be a concern but anyways I'll revert it to 2.3.0.